extensions/sql-database-projects/package-lock.json

```diff
@@ -0,0 +1,6716 @@
+{
```

Comment:
this file comes from running `npm install` and shouldn't be needed since we're using `yarn install`

Reply:
removed